admin_manual/installation/nginx.rst

```diff
@@ -123,13 +123,10 @@ webroot of your nginx installation. In this example it is
       location ^~ /.well-known {
           # The following 6 rules are borrowed from `.htaccess`
       
-          rewrite ^/\.well-known/host-meta\.json  /public.php?service=host-meta-json  last;
-          rewrite ^/\.well-known/host-meta        /public.php?service=host-meta       last;
-          rewrite ^/\.well-known/webfinger        /public.php?service=webfinger       last;
-          rewrite ^/\.well-known/nodeinfo         /public.php?service=nodeinfo        last;
-          
           location = /.well-known/carddav     { return 301 /remote.php/dav/; }
           location = /.well-known/caldav      { return 301 /remote.php/dav/; }
+          # Anything else is dynamically handled by Nextcloud
+          location ^~ /.well-known            { return 301 /index.php$uri; }
```

Comment:
I think `$request_uri` must be used here as `$uri` is the normalised/decided URI with query string and anchor removed, which is not what is wanted here?

Reply:
@daita could that explain why you sometimes didn't see query parameters?!